maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -552,6 +556,7 @@ private fun MarkerImpl(
 @Composable
 @GoogleMapComposable
 public fun AdvancedMarker(
+    contentDescription: String? = "",
```

Comment:
is there a reason not to keep `state` the first parameter? Seems a better choice to me.

Also, I believe this turns the PR into a breaking change.

Reply:
I called it a breaking change because it will break callers that do not use named parameters.